### ISSUE TITLE
Error after upgrading to latest Jitsi

### ISSUE THREAD

**Issue by Dual-0** [systemli/prometheus-jitsi-meet-exporter#72]
Like issue #8 the newest jitsi breaks the exporter.

error:
`json decoding error: json: cannot unmarshal number 0.0 into Go struct field videoBridgeStats.octo_send_bitrate of type int`

curl -s localhost:8080/colibri/stats | jq
```
{
  "inactive_endpoints": 0,
  "inactive_conferences": 0,
  "total_ice_succeeded_relayed": 0,
  "colibri2": true,
  "total_loss_degraded_participant_seconds": 0,
  "bit_rate_download": 0,
  "local_active_endpoints": 0,
  "muc_clients_connected": 1,
  "total_participants": 0,
  "total_packets_received": 0,
  "rtt_aggregate": 0,
  "packet_rate_upload": 0,
  "p2p_conferences": 0,
  "total_aimd_bwe_expirations": 0,
  "total_loss_limited_participant_seconds": 0,
  "preemptive_kfr_suppressed": 0,
  "local_endpoints": 0,
  "octo_send_bitrate": 0,
  "total_dominant_speaker_changes": 0,
  "endpoints_with_spurious_remb": 0,
  "receive_only_endpoints": 0,
  "total_colibri_web_socket_messages_received": 0,
  "octo_receive_bitrate": 0,
  "version": "2.1.617-ga8b39c3f",
  "total_ice_succeeded": 0,
  "total_colibri_web_socket_messages_sent": 0,
  "total_bytes_sent_octo": 0,
  "total_data_channel_messages_received": 0,
  "total_conference_seconds": 0,
  "num_eps_oversending": 0,
  "bit_rate_upload": 0,
  "total_conferences_completed": 0,
  "octo_conferences": 0,
  "num_eps_no_msg_transport_after_delay": 0,
  "endpoints_sending_video": 0,
  "packet_rate_download": 0,
  "muc_clients_configured": 1,
  "outgoing_loss": 0,
  "overall_loss": 0,
  "conference_sizes": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_packets_sent_octo": 0,
  "total_layering_changes_received": 0,
  "total_relays": 0,
  "conferences_by_video_senders": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "endpoints_with_high_outgoing_loss": 0,
  "stress_level": 0,
  "jitter_aggregate": 0,
  "drain": false,
  "total_video_stream_milliseconds_received": 0,
  "total_ice_succeeded_tcp": 0,
  "octo_endpoints": 0,
  "current_timestamp": "2022-01-29 11:49:37.995",
  "total_packets_dropped_octo": 0,
  "num_relays_no_msg_transport_after_delay": 0,
  "conferences": 0,
  "participants": 0,
  "total_keyframes_received": 0,
  "average_participant_stress": 0.01,
  "largest_conference": 0,
  "total_packets_sent": 0,
  "endpoints": 0,
  "total_data_channel_messages_sent": 0,
  "incoming_loss": 0,
  "total_bytes_received_octo": 0,
  "octo_send_packet_rate": 0,
  "conferences_by_audio_senders": [
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_conferences_created": 0,
  "total_ice_failed": 0,
  "preemptive_kfr_sent": 0,
  "threads": 37,
  "videochannels": 0,
  "total_packets_received_octo": 0,
  "graceful_shutdown": false,
  "octo_receive_packet_rate": 0,
  "total_bytes_received": 0,
  "total_loss_controlled_participant_seconds": 0,
  "total_partially_failed_conferences": 0,
  "endpoints_sending_audio": 0,
  "dtls_failed_endpoints": 0,
  "total_bytes_sent": 0,
  "mucs_configured": 1,
  "total_failed_conferences": 0,
  "mucs_joined": 1
}
```

**Comment by 0x46616c6b** [comment]
Can you recheck with the version `1.1.8`?

**Comment by jb-cd** [comment]
I have the same issue. I'm using version 1.1.8. 

We still get: prometheus-jitsi-meet-exporter-linux-amd64[5199]: json decoding error: json: cannot unmarshal number 0.0 into Go struct field videoBridgeStats.octo_send_bitrate of type int

The binary that has been used:
https://github.com/systemli/prometheus-jitsi-meet-exporter/releases/download/1.1.8/prometheus-jitsi-meet-exporter-linux-amd64

sha512sum prometheus-jitsi-meet-exporter-linux-amd64
986a933c6acc9c644f5f340236748ebd83b064f261b3c79977dba6c232b7c3647ebd58a5562396b0ce02e1d75d9061f6a5084cb89761e055a9e515ae4bd2af55  prometheus-jitsi-meet-exporter-linux-amd64

**Comment by 0x46616c6b** [comment]
Can you provide the JSON from your videobridge? So we could adjust the Exporter and add the JSON to tests to prevent regressions.

**Comment by jb-cd** [comment]
curl -s localhost:8080/colibri/stats | jq
```
{
  "inactive_endpoints": 0,
  "inactive_conferences": 0,
  "total_ice_succeeded_relayed": 0,
  "colibri2": true,
  "total_loss_degraded_participant_seconds": 1675,
  "bit_rate_download": 6837.841,
  "local_active_endpoints": 2,
  "muc_clients_connected": 1,
  "total_participants": 58,
  "total_packets_received": 10813083,
  "rtt_aggregate": 23.159109,
  "packet_rate_upload": 732,
  "p2p_conferences": 0,
  "total_aimd_bwe_expirations": 0,
  "total_loss_limited_participant_seconds": 1495,
  "preemptive_kfr_suppressed": 1081,
  "local_endpoints": 2,
  "octo_send_bitrate": 0,
  "total_dominant_speaker_changes": 1284,
  "endpoints_with_spurious_remb": 1,
  "receive_only_endpoints": 0,
  "total_colibri_web_socket_messages_received": 60864,
  "octo_receive_bitrate": 0,
  "version": "2.1.617-ga8b39c3f",
  "total_ice_succeeded": 50,
  "total_colibri_web_socket_messages_sent": 86618,
  "total_bytes_sent_octo": 0,
  "total_data_channel_messages_received": 0,
  "total_conference_seconds": 16729,
  "num_eps_oversending": 0,
  "bit_rate_upload": 5375.254,
  "total_conferences_completed": 16,
  "octo_conferences": 0,
  "num_eps_no_msg_transport_after_delay": 1,
  "endpoints_sending_video": 2,
  "packet_rate_download": 949,
  "muc_clients_configured": 1,
  "outgoing_loss": 0,
  "overall_loss": 0,
  "conference_sizes": [
    0,
    0,
    1,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_packets_sent_octo": 0,
  "total_layering_changes_received": 30,
  "total_relays": 0,
  "conferences_by_video_senders": [
    0,
    0,
    1,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "endpoints_with_high_outgoing_loss": 0,
  "stress_level": 0.03308,
  "jitter_aggregate": 3.327188334292373,
  "drain": false,
  "total_video_stream_milliseconds_received": 96520790,
  "total_ice_succeeded_tcp": 0,
  "octo_endpoints": 0,
  "current_timestamp": "2022-02-24 13:15:05.995",
  "total_packets_dropped_octo": 0,
  "num_relays_no_msg_transport_after_delay": 0,
  "conferences": 1,
  "participants": 2,
  "total_keyframes_received": 11013,
  "average_participant_stress": 0.01,
  "largest_conference": 2,
  "total_packets_sent": 21352431,
  "endpoints": 2,
  "total_data_channel_messages_sent": 0,
  "incoming_loss": 0,
  "total_bytes_received_octo": 0,
  "octo_send_packet_rate": 0,
  "conferences_by_audio_senders": [
    0,
    0,
    1,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0,
    0
  ],
  "total_conferences_created": 17,
  "total_ice_failed": 0,
  "preemptive_kfr_sent": 2,
  "threads": 61,
  "videochannels": 2,
  "total_packets_received_octo": 0,
  "graceful_shutdown": false,
  "octo_receive_packet_rate": 0,
  "total_bytes_received": 6032174393,
  "total_loss_controlled_participant_seconds": 38871,
  "total_partially_failed_conferences": 0,
  "endpoints_sending_audio": 2,
  "dtls_failed_endpoints": 0,
  "total_bytes_sent": 10945229788,
  "mucs_configured": 1,
  "total_failed_conferences": 0,
  "mucs_joined": 1
}
```

**Comment by 0x46616c6b** [comment]
I did a fix test with your provided JSON and could not reproduce an error. Do you really sure that you use the correct version of the exporter?

**Comment by 0x46616c6b** [comment]
I rechecked again and noticed that in the release 1.1.8 old assets was uploaded. Can you download the release from the page again and try it out? Also a fixed release (1.1.9) ist out.

**Comment by jb-cd** [comment]
Thank you very much the issue has been solved.
